modules/accounts/email.nix

```diff
@@ -498,8 +498,12 @@ in {
         if hasPrefix "/" p then p else "${config.home.homeDirectory}/${p}";
       description = ''
         The base directory for account maildir directories. May be a
-        relative path, in which case it is relative the home
-        directory.
+        relative path (e.g. the user setting this value as "MyMaildir"),
+        in which case it is relative the home directory (e.g. resulting
+        in "~/MyMaildir"). Note that "$HOME/Maildir" being cited as the
+        default value is showing the result, and could be achieved by
+        the user setting this value as "Maildir". Using "$HOME/" will
+        not do any interpolation.
```

Comment:
I think the latter two sentences could be removed if we just fix the `defaultText` field to match reality. Something like

``` nix
defaultText = literalExpression ''"''${config.home.homeDirectory}/Maildir"'';
```

Alternatively simply

``` nix
defaultText = "Maildir";
```

Reply:
I've made the adjustments in the latest commits.